app/controllers/authenticate_controller.rb

```diff
@@ -214,4 +222,8 @@ def installed_authenticators
   def enabled_authenticators
     Authentication::InstalledAuthenticators.enabled_authenticators(ENV)
   end
+
+  def encoded_response?
+    request.accept_encoding&.include?("base64")
```

Comment:
```suggestion
    request.accept_encoding&.casecmp?("base64")
```
I think it should use `casecmp` because you want to check specifically that it is equal to `base64` (case is insensitive), not that it contains it, among other characters.